manager/src/main/java/org/openremote/manager/map/MapService.java

```diff
@@ -196,9 +232,20 @@ public void setData() throws ClassNotFoundException, SQLException, NullPointerEx
         if (!mapTilesPath.toFile().exists()) {
             return;
         }
+        Path customMapTilesPath = getCustomMapDataPath();
+        if (customMapTilesPath != null) {
+            if (delete) {
+                connection.close();
```

Comment:
try catch needed and should check != null

Reply:
Don't think this should be conditional on delete any change to the connection var should first close old connection